### ISSUE TITLE
Pulse Shotgun missing smoke canister model on alt fire

### ISSUE THREAD

**Issue by Silverfang22** [JKGDevs/JediKnightGalaxies#304]
Need to investigate this one

**Comment by DarthFutuza** [comment]
Also just FYI.  This also causes the game to assert as well if run in a debug build because of this check, which appears to just mean the model can't be found.
```cpp
//tr_scene.cpp line 239
#ifdef _DEBUG
	if (ent->reType == RT_MODEL)
	{
		assert(ent->hModel || ent->ghoul2 || ent->customShader);
	}
#endif```

**Comment by Silverfang22** [comment]
Should be fixed with aad29b2d3855ea4f9409f2eb457c5fe25d9f0e83
